### ISSUE TITLE
angular-cli@1.0.0-bata.11-webpack.2 (RC5 starter pack) "--proxy option" stop working

### ISSUE THREAD

**Issue by jacovc86** [angular/angular-cli#1712]
window 10

angular-cli: 1.0.0-beta.11-webpack.2
node: 6.3.1
os: win32 x64

Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
create a new project and serve it with ng serve --proxy http://some/proxy/url

Worked in the rc4 project structure

**Comment by mmrath** [comment]
Please see PR https://github.com/angular/angular-cli/pull/1487

**Comment by mmrath** [comment]
Duplicate of #889

**Comment by jacovc86** [comment]
Thanks @mmrath, found a temporary hack:

angular-cli/addons/ng2/tasks/ng-serve-webpack.ts:33

...
inline: true,
proxy: {
   '/rest': commandOptions.proxy
 }

if anyone else just want to get it going until it is configurable

**Comment by filipesilva** [comment]
Closed by https://github.com/angular/angular-cli/pull/1487

**Comment by wz2b** [comment]
This workaround works, thanks for the reference.  In my case I need an additional option as well:

```
      proxy: {
         '/api':  {
             target: 'https://my.real.server.com,
             secure: false
         }
      }
```

It would be nice that, once this is permanently solved, there would be some way to pass in more robust options than just a string - possibly even a place to add a function/closure to be able to do some simple request mapping

**Comment by jasonswett** [comment]
@filipesilva This might be a dumb question but if this issue is closed by #1487, how come proxy doesn't work on the latest Webpack version (`1.0.0-beta.11-webpack.8`)?

**Comment by mmrath** [comment]
Please see https://github.com/angular/angular-cli/pull/1896 for documentation on how to use the --proxy-config option.
